appveyor-windows-build-go.yml

```diff
@@ -0,0 +1,79 @@
+version: 1.0.{build}
```

Comment:
Did you create a appveyor project for this? Can you add it to the repo so we can see the build results from the github UI?

Reply:
O it is named: `continuous-integration/appveyor/pr`. Can you update the name in appveyor so the name we see here explains which tests that build belongs too?